app/controllers/qa/issues_controller.rb

```diff
@@ -46,11 +46,8 @@ def multiple_update
             redirect_to project_issues_path(current_project), notice: 'State updated successfully.'
           end
         end
-
-        format.json { head :ok }
```

Comment:
did you do a sweep to see anywhere else where we can remove unnecessary `format.json`s?

Reply:
It doesn't look like there's other unused format.jsons